### ISSUE TITLE
0.17.2 crashes when pressing `Enter` on editor

### ISSUE THREAD

**Issue by pablo-abc** [seek-oss/playroom#151]
After loading playroom and starting with no issue, if you were to press enter when there's an element, the whole app will crash with the following error on Firefox:

```
The above error occurred in the <StoreProvider> component:
    in StoreProvider

Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://fb.me/react-error-boundaries to learn more about error boundaries. 

TypeError: testCode[line] is undefined
```

And the following on Chrome:

```
Uncaught TypeError: Cannot read property 'slice' of undefined
    at insertAtCursor (cursor.ts?6ed8:17)
    at isValidLocation (cursor.ts?6ed8:33)
    at eval (StoreContext.tsx?302f:177)
    at updateReducer (react-dom.development.js?61bb:15157)
    at Object.useReducer (react-dom.development.js?61bb:15954)
    at useReducer (react.development.js?72d0:1501)
    at StoreProvider (StoreContext.tsx?302f:387)
    at renderWithHooks (react-dom.development.js?61bb:14825)
    at updateFunctionComponent (react-dom.development.js?61bb:17058)
    at beginWork (react-dom.development.js?61bb:18643)

The above error occurred in the <StoreProvider> component:
    in StoreProvider
```

I'm not really sure if any of this info is of help to you. 😅

**Comment by michaeltaranto** [comment]
Thanks mate, I have an idea, will have a look

**Comment by michaeltaranto** [comment]
Should be fixed in `v0.17.3`. Thanks for the quick report.
